Tiltfile

```diff
@@ -1,12 +1,17 @@
 # -*- mode: Python -*-
 
+envsubst_cmd = "./hack/tools/bin/envsubst"
+kubectl_cmd = "/usr/bin/kubectl"
+
 load("ext://uibutton", "cmd_button", "text_input")
 
 # set defaults
 version_settings(True, ">=0.22.2")
 
 settings = {
     "deploy_cert_manager": True,
+    "kubernetes_version": "v1.22.6",
```

Comment:
I would prefer to avoid having a version in the Tiltfile, can this come from the kustomize_substitutions?